src/messaging/tests/echo/echo_responder.cpp

```diff
@@ -85,13 +87,15 @@ int main(int argc, char * argv[])
 
     if (useTCP)
     {
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
         err = gTCPManager.Init(chip::Transport::TcpListenParameters(chip::DeviceLayer::TCPEndPointManager())
                                    .SetAddressType(chip::Inet::IPAddressType::kIPv6));
         SuccessOrExit(err);
 
         err = gSessionManager.Init(&chip::DeviceLayer::SystemLayer(), &gTCPManager, &gMessageCounterManager, &gStorage,
                                    &gFabricTable, gSessionKeystore);
         SuccessOrExit(err);
+#endif // INET_CONFIG_ENABLE_TCP_ENDPOINT
```

Comment:
Does there need to a #else here if someone tries to run the TCP test with no TCP support compiled in and have that either error out or print a warning message?

Reply:
Have folded in the `useTCP` flag inside INET_CONFIG_ENABLE_TCP_ENDPOINT.